### PR TITLE
Add UPLOADING to lsif upload state

### PR DESCRIPTION
**Pull request by efritz**

### CODE REVIEW

File: web/src/enterprise/repo/settings/RepoSettingsCodeIntelligencePage.tsx [hunk 1]
```diff
@@ -78,7 +78,9 @@ const LsifUploadNode: FunctionComponent<LsifUploadNodeProps> = ({ node, onDelete
                         -
                         <span className="ml-2">
                             <Link to={`./code-intelligence/lsif-uploads/${node.id}`}>
-                                {node.state === GQL.LSIFUploadState.PROCESSING ? (
+                                {node.state === GQL.LSIFUploadState.UPLOADING ? (
+                                    <span>Uplaoding</span>
```
Comment (eseliger) [inline web/src/enterprise/repo/settings/RepoSettingsCodeIntelligencePage.tsx:82]:
```suggestion
                                    <span>Uploading</span>
```

**Comment by efritz** [comment]
> isn't a backend change required here as well?

which backend? This already landed in the release branch and I forgot to update the uses of the enum values.

**Comment by efritz** [comment]
See https://github.com/sourcegraph/sourcegraph/pull/10456.

**Comment by eseliger** [comment]
I see, cool :)

**Pull request merged**